education.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My education details</title>
+	<meta charset="utf-8">
+	<meta name="Description" content=This page show my education details">
+	<meta name="keywords" content="nazish, ABSS Institute of Technology, SBSS College, Education">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+	    <h2>My Education Details.</h2>
+	    <table>

```

Comment:
Absolutely awesome job with the table!

After you make some small changes, we'll merge this in and add in some improved accessibility. But this is great work.